### PR TITLE
Feature/cart

### PR DESCRIPTION
**Pull request by Nandiniatri**
Cart

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Logs
```
Comment (rajputankit91) [inline .gitignore:1]: why are we changing git ignore in this branch

Comment (SATYAM08RAJPUT) [inline .gitignore:1]: **why are we changing in gitignore in this branch**

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
-# Component-Library
\ No newline at end of file
+# React + Vite
```
Comment (rajputankit91) [inline README.md:1]: same as above ?

File: eslint.config.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import js from '@eslint/js'
```
Comment (rajputankit91) [inline eslint.config.js:1]: same as above ?

**Pull request merged**